### DISCUSSION TITLE
From where to get tarball

### DISCUSSION THREAD

**Discussion by awasthan** [ppc64le/build-scripts#909] [Q&A]
Hi,
   From where can I get tarball 'instana-agent-linux-ppcle-64bit.tar.gz' ?
@amitsadaphule Can you please confirm?

Thanks

**Comment by amitsadaphule** [comment]
Hi,

You can either register for a trial account on instana.com or request for a dedicated IBM account (you can follow the steps from the doc that you're pointed to when you try to register for free trial on instana.com using IBM id). Once you login, go to

More > Agents > Installing Instana Agents > (OS) Linux > (Linux) Technology > Archive (tar.gz) > Linux (64-Bit PowerPC Little Endian) > Download.

Regards,
Amit

**Reply by gerrith3** [reply]
@amitsadaphule Should this info be in the README for this package?

**Reply by amitsadaphule** [reply]
Actually, the tarball in question here, is for the agent and not the operator (which we have ported). So, I wasn't sure whether to include the details on where to get that. I can add the first part of the info above, but I feel adding the UI navigation wouldn't be a good idea, since that may change.

**Reply by awasthan** [reply]
Thanks Amit!

**Reply by awasthan** [reply]
@amitsadaphule When we create the instana agent custom resource, it creates a daemonset. This daemonset has the agent image name as 'instana/agent' and imagePullPolicy as 'Always' hard-coded. Can these two be passed from the agent CR so that I can specify to pull the image from a my private registry and change the imagePullPolicy also?
Since I am deploying in an air-gap environment; so need to control both these vaules. Changing in the daemonset is not the appropriate solution as the operator reconciles the values.

**Reply by amitsadaphule** [reply]
@awasthan I think you'll need to update/add to [this](https://github.com/ppc64le/build-scripts/blob/master/i/instana-agent-operator/instana-agent-operator_rhel8.2.sh#L70-L75) part of the script for specifying your private registry. I guess the following lines will need to be updated for private registry:
https://github.com/instana/instana-agent-operator/blob/main/src/main/resources/instana-agent.daemonset.yaml#L29
https://github.com/instana/instana-agent-operator/blob/main/olm/operator-resources/instana-agent-operator.yaml#L218

And regarding the `imagePullPolicy`, it's already updated to `IfNotPresent` in [this](https://github.com/ppc64le/build-scripts/blob/master/i/instana-agent-operator/instana-agent-operator_rhel8.2.sh#L70-L75) block.

**Reply by awasthan** [reply]
Thank you
